libheif/plugins/heif_encoder_svt.cc

```diff
@@ -624,7 +624,7 @@ struct heif_error svt_encode_image(void* encoder_raw, const struct heif_image* i
   // disable 2-pass
   svt_config.rc_stats_buffer = (SvtAv1FixedBuf) {nullptr, 0};
 
-  svt_config.rate_control_mode = 0; // constant QP
+  svt_config.rate_control_mode = 0; // constant rate factor
```

Comment:
Note that line 622 is commented out:

```
  //svt_config.enable_adaptive_quantization = 0;
```

Therefore `svt_config.enable_adaptive_quantization` has the default value of 2.